R/labels.R

```diff
@@ -45,29 +45,33 @@ get_labels <- function(p, aes = NULL) {
 #' Note that `uses_labels()` will match [`NULL`] if a label is explicitly set to
 #' [`NULL`] ***or*** if a requested aesthetic is not present in the plot.
 #'
-#' @param p A ggplot object
-#' @param ... <[`dynamic-dots`][rlang::dyn-dots]> Named [character] strings.
-#'   Each argument should have a name matching a [ggplot][ggplot2::ggplot]
-#'   [aesthetic][ggplot2::aes] or [label][ggplot2::labs],
-#'   and a value matching the expected label.
-#'   Named character strings may be passed as arguments or as list elements.
-#'
-#' @return A logical vector of the same length as the number of inputs to `...`.
-#'
 #' @family functions for checking labels
 #' @export
```

Comment:
Can we move these two lines to the very bottom of the documentation, just before the function definition?

Reply:
Resolved in https://github.com/rstudio/ggcheck/pull/27/commits/a6c9305d17a1d2bfba7828de8e1622f935189a3a